### PR TITLE
add tests for users and update jenkins file

### PR DESCRIPTION
**Pull request by nberard**
add tests for users and update jenkins file

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -8,7 +8,9 @@ stage("Unit tests") {
         wrap([$class: 'AnsiColorBuildWrapper']) {
             sh '''
             cp config.py.dist config.py
-            docker run -v $(pwd):/srv/spark-stat-analyzer par-vm232.srv.canaltp.fr:5000/stat-analyser:test
+            docker pull par-vm232.srv.canaltp.fr:5000/stat-analyser:test
+            docker run --rm -v $(pwd):/srv/spark-stat-analyzer par-vm232.srv.canaltp.fr:5000/stat-analyser:test sh -c './run_test.sh'
+            #junit 'junit.xml'
```
Comment (vincentlepot) [inline Jenkinsfile]: Why keep this commented?

File: tests/fixtures/users.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{"user_id": 15, "request_date":1484467930,"user_name":"Bobby"}
+{"user_id": 15, "request_date":1484489770,"user_name":"Bobby new name"}
+{"user_id": 15, "request_date":1484477830,"user_name":"Bobby"}
+{"user_id": 42, "request_date":1484459770,"user_name":"Kenny"}
+{"user_id": 42, "request_date":1484467830,"user_name":"Kenny mid name"}
+{"user_id": 42, "request_date":1484467930,"user_name":"Kenny last name"}
+{"user_id": 666, "request_date":1484467930,"user_name":"Billy"}
+{"user_id": 666, "request_date":1484468930,"user_name":"Billy"}
+{"user_id": 666, "request_date":1484469930,"user_name":"Billy"}
```
Comment (azime) [inline tests/fixtures/users.json]: users.log, No ?

File: tests/users_sql_test.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+def test_users_sql(spark):
+    file_list = 'tests/fixtures/users.json'
+    df = common.get_sql_data_frame(spark, file_list)
+    results = users_sql.process(df)
+    expected_results = [Row(user_id=15, last_user_name=u'Bobby new name', first_date=1484467930), Row(user_id=42, last_user_name=u'Kenny last name', first_date=1484459770), Row(user_id=666, last_user_name=u'Billy', first_date=1484467930)]
+    # Row created element lose order so we can't compare strictly with assert ==
+    assert len(results) == len(expected_results)
+    for row in results:
+        for expected_row in expected_results:
+            if row.user_id == expected_row.user_id:
```
Comment (azime) [inline tests/users_sql_test.py]: if row.user_id != expected_row.user_id ?

**Comment by azime** [comment]
Management of the database should be reviewed

**Pull request merged**